src/components/NavigationTab/NavigationTab.svelte

```diff
@@ -26,10 +28,11 @@
 </script>
 
 <div class="flex w-full">
-	{#each tabs.filter((tab) => tab.show) as { id, label } (id)}
+	{#each tabs.filter((tab) => tab.show) as { id, label, disabled } (id)}
 		<button
-			class={activeTab === id ? 'tab-active' : 'tab-inactice'}
+			class={activeTab === id ? `tab-active text-${align}` : `tab-inactive text-${align}`}
```

Comment:
อันนี้เป็น catch ของ tailwind ครับ ห้าม substitute tailwind class เพราะว่า Tailwind จะอ่าน source code เราตอน build time และเตรียม CSS class ให้เฉพาะเท่าที่มันเจอครับ `text-${align}` ไม่ใช่ valid tailwind class และ tailwind ก็ไม่รู้ว่า align เป็นค่าอะไรได้บ้าง
- ตอน dev ใช้ได้เพราะ tailwind จะใส่มาให้ทั้งหมดเพื่อความเร็วของ hot reload
- ตอน build ถ้าบางเอิญใช้ได้ แสดงว่ามี code อื่นที่ใช้ class ที่เราต้องการพอดี
cc: @vzsky 
Ref: https://tailwindcss.com/docs/detecting-classes-in-source-files#dynamic-class-names

อีกอย่างคือ text-align ใช้ซ้ำสองรอบ svelte ให้แทรก condition ใน double quote ตรงไหนก็ได้ครับ

```svelte
<div
  class="someclassname {dynamicClassName} {activeTab === id ? 'tab-active' : 'tab-inactive'}"
>
```